accounts/management/commands/fill_downloads.py

```diff
@@ -19,10 +19,11 @@
 #
 
 import logging
+
 from django.core.management.base import BaseCommand
 from django.db import connection
 
-logger = logging.getLogger("console")
+console_logger = logging.getLogger("console")
 
 
 class Command(BaseCommand):
```

Comment:
You don't use the Logging command here. Is there a reason why?

Reply:
Not really. Updated here fcaac4a31ad0d681e020aea7d1255856712cca3e